mocks/todos.ts

```diff
@@ -0,0 +1,54 @@
+import { TodoItem } from 'types/todos'
```

Comment:
Let's move this file to the corresponding module.

Reply:
Fixed, thanks!